app/src/api/api.js

```diff
@@ -48,10 +48,20 @@ export const replyMessage = (message, messageConversationId) =>
         throw error;
       });
 
+export const deleteMessageConversation = messageConversationId =>
+  getD2Instance()
+    .then(instance =>
+      instance.Api.getApi().delete(`messageConversations/${messageConversationId}`))
+    .then(result => result)
+    .catch(error => {
+      throw error;
```

Comment:
You should wrap your error in a new error when you re-throw to maintain the stacktrace.

`throw new Error(error)`